Package.swift

```diff
@@ -44,80 +43,68 @@ let package = Package(
     .target(
       name: "Database",
       dependencies: [
-        "Either",
         "GitHub",
-        "Logging",
         "Models",
-        "PostgreSQL",
-        "Prelude",
         "Stripe",
-        "Tagged",
+        .product(name: "Either", package: "swift-prelude"),
```

Comment:
Swift 5.2 will require describing external packages now. I think it improves resolution, and it's nice as documentation.

Reply:
nice!